Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3656,6 +3656,10 @@ script:
     - description: The name of the virtual system to be configured. If no vsys is
         mentioned, this command will not use the vsys parameter.
       name: vsys
+    - description: The device group to target
+      name: device-group
+    - description: Return raw XML
+      name: is_xml
```

Comment:
@GuyAfik typically we will use `raw-response=true` to show the API results for these types of commands.  When saving to context you can simply use the `extend-context=PanOS=.` argument.  Not needed for this PR though since this functionality is passed under the hood and not used directly by the end user.